### ISSUE TITLE
Dataloader folder -missing

### ISSUE THREAD

**Issue by asil07** [SH-Diao123/MSAN-CNN#1]
Can you share the data-loader file and briefly explain the difference four path below

>     a_path = '' " 
>     b_path = ''  "
>     aa_path = '' "
>     bb_path = '' "

**Comment by SH-Diao123** [comment]
This represents the images and labels of training data and validation data, respectively

**Comment by asil07** [comment]
Thank you for your prompt response. We kindly request a brief 10-minute Zoom meeting with you. The purpose of this meeting is to discuss pertinent inquiries regarding your paper, which is currently being studied within our research laboratory based in South Korea. We greatly appreciate your anticipated cooperation.

**Comment by HAOZHAO1** [comment]
Thank you so much for sharing，Can you share the code you wrote？“from data_loader import myImageFloder, myImageFloderAttn”——myImageFloderAttn??

**Comment by SH-Diao123** [comment]
> Thank you so much for sharing，Can you share the code you wrote？“from data_loader import myImageFloder, myImageFloderAttn”——myImageFloderAttn??

of cause. It is just a simple Dataset Class. But the code is on the intranet, and I haven't been able to connect recently. Just give me some times to find the code if you need it

**Comment by HAOZHAO1** [comment]
Of course, I'll be glad to hear from you,Thanks again.














------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "SH-Diao123/MSAN-CNN"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年6月1日(星期四) 中午12:26
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [SH-Diao123/MSAN-CNN] Dataloader folder -missing (Issue #1)





  
Thank you so much for sharing，Can you share the code you wrote？“from data_loader import myImageFloder, myImageFloderAttn”——myImageFloderAttn??
  
of cause. It is just a simple Dataset Class. But the code is on the intranet, and I haven't been able to connect recently. Just give me some times to find the code if you need it
